logic/concludable/ConjunctionConcludable.java

```diff
@@ -170,9 +169,6 @@ public Isa(final IsaConstraint constraint) {
 
         @Override
         public Stream<Unification> unify(ThenConcludable.Isa headIsa) {
-            Set<Label> possibleLabels = constraint().getTypeHints();
-            Set<Label> otherLabels = headIsa.constraint().getTypeHints();
-            possibleLabels.retainAll(otherLabels);
             return Stream.of(new Unification(this, headIsa, null)); // TODO Add variable mapping
```

Comment:
this is properly repalced by your other PR right?

Reply:
Indeed.